sql_metadb/report_queries/erm/count/erm_agreement_count.sql

```diff
@@ -43,24 +42,27 @@ FROM
     LEFT JOIN folio_derived.agreements_subscription_agreement_entitlement AS sa_ent ON pci_list.entitlement_id = sa_ent.entitlement_id
     LEFT JOIN folio_agreements.refdata_value AS agrestat ON agrestat.rdv_id = sa_ent.subscription_agreement_status
 
-WHERE 
-	((agrestat.rdv_label = (SELECT agreement_status FROM parameters)) OR 
-		((SELECT agreement_status FROM parameters) = '')
-/*	
- -- As there is an issue on the field type for dates in MetaDB date filters are commented out and will be reimplemented when solved.
-	AND 			
+WHERE
+ -- Don't count removed titles from packages
+    pci_list.pci_removed_ts IS NULL
+ -- Start with parameter filters
+    AND
+	((agrestat.rdv_label = (SELECT agreement_status FROM parameters)) OR
+		((SELECT agreement_status FROM parameters) = ''))
+	AND
     ((pci_list.pci_access_start < (SELECT pci_end_date FROM parameters) AND
 	    (pci_list.pci_access_end >= (SELECT pci_start_date FROM parameters) OR pci_list.pci_access_end IS NULL))
-	    OR 
-		(((SELECT pci_start_date FROM parameters) IS NULL) 
+	    OR
+		(((SELECT pci_start_date FROM parameters) IS NULL)
 			OR ((SELECT pci_end_date FROM parameters) IS NULL)))
```

Comment:
Last bracket too much.

Reply:
Last bracket is to include the first part of parenthesis from line 53 on